### DISCUSSION TITLE
using this extension inside a devcontainer

### DISCUSSION THREAD

**Discussion by Lobosque** [microsoft/vscode-docker#3461] [Q&A]
In this case, the docker daemon is running on the host, so the extension does not detect docker.

Any way to configure it in a way it connects to the host docker?

**Comment by karolz-ms** [comment]
The extension should be usable from a devcontainer, you just need to make sure the Docker socket is properly mapped into the container, and you need to have Docker CLI installed. See https://code.visualstudio.com/docs/remote/containers#_how-can-i-build-or-deploy-container-images-into-my-local-docker-kubernetes-install-when-working-inside-a-container for more information. Hope this helps!
